### DISCUSSION TITLE
I ran qutebrowser as root but it didn't open..will there be a problem ?

### DISCUSSION THREAD

**Discussion by ReductoAdAbsurdum** [qutebrowser/qutebrowser#5529] [Q&A] [answered]
I mistakenly ran qutebrowser as root...however it gave me an error and it did not launch...

will that cause problems ? like permission problems ?

**Comment by The-Compiler** [comment, marked as answer]
This could happen because what's failing is actually something inside QtWebEngine/Chromium, after qutebrowser already initialized e.g. storage directories.

You should be able to double-check with something like this in a shell (as user):

```
find ~/.local/share/qutebrowser ~/.config/qutebrowser ~/.cache/qutebrowser -not -uid $UID
```

If that outputs any files, those files are owned by something other than your current user. See `:version` to get the exact locations of those directories, the ones listed are the default locations on Linux.

**Reply by ReductoAdAbsurdum** [reply]
by user you mean a non-root user   ?
I purged qutebrowser and re-installed it...I guess that takes care of the issue...

Many Thanks

**Comment by DedSecer** [comment]
@The-Compiler 
I've check the command  `find ~/.local/share/qutebrowser ~/.config/qutebrowser ~/.cache/qutebrowser -not -uid $UID`,and it don't outputs any files.I still cannot run it.
Here are the output atfer I run qutebrowser:
`[4129:4129:0703/203339.209876:ERROR:zygote_host_impl_linux.cc(89)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.`
So I try to run`qutebrowser --no-sandbox`,but it seem that qutebrowser doesn't support this argument.
How to solve this?
(Sorry for my poor English)

**Reply by The-Compiler** [reply]
@DedSecer If you're getting this error, you're still running qutebrowser as root.

**Reply by DedSecer** [reply]
Sorry,I think my expression was wrong.
My problem is 'how to run qutebrowser as root?'

**Reply by The-Compiler** [reply]
Don't. Why would you want to?

**Reply by DedSecer** [reply]
I want to custom a archlinux installer(archiso),it will login as root by default,and I want a browser for me to look for some imformation during I installing linux(or do some other things).

**Reply by The-Compiler** [reply]
So... why not create a user and run the browser under that user? FWIW that error message is coming from Chromium, and you can pass that `--no-sandbox` argument to Chromium by running with `--qt-flag no-sandbox`. However, at that point you're running a web browser, as root, without any sandboxing - which isn't exactly a good idea.

**Reply by DedSecer** [reply]
OK.Thanks a lot!
